app/views/layouts/application.html.erb

```diff
@@ -33,7 +33,16 @@
         </div>
       </main>
     </div>
-    <%= render "govuk_publishing_components/components/layout_footer" %>
+    <%= render "govuk_publishing_components/components/layout_footer", {
+      meta: {
+        items: [
+          {
+            href: "/privacy",
+            text: "Privacy notice"
```

Comment:
tiny nit: could be a translatable string

Reply:
Yeah, go on then, let's be consistent with everything else.

Reply:
Done!